Sources/MuxUploadSDK/PublicAPI/Version.swift

```diff
@@ -11,9 +11,9 @@ public struct Version {
   /// Major version.
   public static let major = 0
   /// Minor version.
-  public static let minor = 2
+  public static let minor = 3
```

Comment:
Thanks for remembering to change these, we should look into adding a script for automating this change

Reply:
I wonder if swift-syntax could help us